foundations/javascript_basics/variables_and_operators.md

```diff
@@ -1,86 +1,129 @@
 ### Introduction
 
-In the previous sections you learned how to structure webpages with HTML and style them with CSS. The next step is to make the webpage *interactive*, which is exactly what JavaScript is for. In this section, we will focus on the fundamentals of JavaScript and how you can use it to manipulate all the various interactions between the web page and user.
+In the previous sections you learned how to structure webpages with HTML and style them with CSS. The next step is to make the webpage *interactive*, which is exactly what JavaScript is for.
+
+In this section, we will focus on the fundamentals of JavaScript and how you can use it to manipulate all the various interactions between the web page and user.
 
 ### Lesson overview
 
 This section contains a general overview of topics that you will learn in this lesson.
 
-- How do you declare a variable?
-- What are three different ways to declare a variable?
-- Which one should you use when?
-- What are the rules for naming variables?
-- What are operators, operands, and operations?
-- What is concatenation and what happens when you add numbers and strings together?
-- What are the different types of operators in JavaScript?
-- What is the difference between == and ===?
-- What are operator precedence values?
-- What are the increment/decrement operators?
-- What is the difference between prefixing and postfixing them?
-- What are assignment operators?
-- What is the Unary Plus Operator?
+- Running JavaScript code using an HTML file and via the browser console.
+- Declaring variables with `let` and `const`.
+- Performing arithmetic with numbers and variables.
+- Operator precedence.
 
 ### How to run JavaScript code
 
-All JavaScript we will be writing in the majority of the Foundations course will be run via the browser. Later lessons in Foundations and the NodeJS path will show you how to run JavaScript outside of the browser environment. Outside of these lessons, for now you should always default to running your JavaScript in the browser unless otherwise specified, otherwise you may run into unexpected errors.
+All JavaScript we will be writing in the majority of the Foundations course will be run via the browser. Later lessons in Foundations and the NodeJS path will show you how to run JavaScript outside of the browser environment.
+
+Outside of these lessons, for now you should always default to running your JavaScript in the browser unless otherwise specified, otherwise you may run into unexpected errors.
 
-The simplest way to get started is to create an HTML file with the JavaScript code inside of it.  Type the basic HTML skeleton into a file on your computer somewhere:
+The simplest way to get started is to create an HTML file with the JavaScript code inside of it. Use the VS Code snippet <kbd>!</kbd> + <kbd>TAB</kbd> to create the basic HTML skeleton in a file on your computer somewhere. Be sure to include the `<script>` tag:
```

Comment:
I'd remove the VSCode bit, let's not assume everyone is using vscode. You could mention that many code editors have keyboard shortcuts to quickly insert this template.

Reply:
The course explicitly teaches learners about VSCode and also at several other points explicit instructions for VSCode specifically are given. Imo it makes sense to give VSCode specific instructions here, if some people want to use other editors they can look up how to add the template there. Similar to how Odin Project only gives instructions for google chrome and not firefox/brave etc. 

Ofc that's just my 2 cents and if you feel like it would be better to not assume the code editor I'll make the necessary changes!

Reply:
You're correct. I didn't realize we made that change.